packages/builders/scripts/vats/init-orchestration.js

```diff
@@ -0,0 +1,20 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+/** @type {import('@agoric/deploy-script-support/src/externalTypes.js').ProposalBuilder} */
+export const defaultProposalBuilder = async ({ publishRef, install }) =>
+  harden({
+    sourceSpec: '@agoric/vats/src/proposals/orchestration-proposal.js',
+    getManifestCall: [
+      'getManifestForOrchestration',
+      {
+        orchestrationRef: publishRef(
+          install('@agoric/vats/src/vat-orchestration.js'),
```

Comment:
how intentional is it that `vat-orchestration` live in `vats` package instead of `orchestration`?